subsys/nrf_security/src/drivers/cracen/ext/cracenpsa/src/key_derivation.c

```diff
@@ -22,9 +22,6 @@
 #include <zephyr/sys/__assert.h>
 #include <zephyr/sys/byteorder.h>
 
-#define PSA_ALG_IS_PBKDF2(kdf_alg)                                                                 \
-	(PSA_ALG_IS_PBKDF2_HMAC(kdf_alg) || (kdf_alg == PSA_ALG_PBKDF2_AES_CMAC_PRF_128))
-
```

Comment:
"got by accident"?

got removed by accident you mean?

It was merged in from HCS because HCS doesn't have this define in it's PSA core.

Now that CRACEN is using a PSA Core that provides this symbol we can remove it :)